ui/raidboss/data/04-sb/alliance/ridorana_lighthouse.js

```diff
@@ -561,12 +571,14 @@
     },
     {
       id: 'Ridorana Yiazmat Magnetic Positive',
-      regex: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of Magnetic Lysis \+ from/,
-      regexDe: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of Positives Magnetfeld from/,
-      regexFr: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of Charge Positive from/,
-      regexJa: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of 磁場崩壊【＋】 from/,
+      regex: Regexes.gainsEffect({ effect: 'Magnetic Lysis \+' }),
+      regexDe: Regexes.gainsEffect({ effect: 'Positives Magnetfeld' }),
+      regexFr: Regexes.gainsEffect({ effect: 'Charge positive' }),
+      regexJa: Regexes.gainsEffect({ effect: '磁場崩壊【＋】' }),
+      regexCn: Regexes.gainsEffect({ effect: '磁场崩坏 正极' }),
+      regexKo: Regexes.gainsEffect({ effect: '자기장 붕괴[\+]' }),
```

Comment:
If these are real brackets, you need to escape them with two backslashes.

Reply:
done